python/test/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+def pytest_collection_modifyitems(items):
+    """
+    Dynamically mark tests based on their file name:
+    - *_test.py: main test (always exercise)
+    - *_extratest.py: exercised only when testing with ffmpeg
+    - *_libtest.py: exercised only to test testlib
+    """
+    for item in items:
+        if "_extratest" in item.nodeid:
+            item.add_marker(pytest.mark.extra)
```

Comment:
This here applies to `pytest` only, it simply systematically marks all tests based on their filename

One can still use `unittest`, this will have no effect on that